src/PaymentSlip.php

```diff
@@ -1453,10 +1453,27 @@ public function getAllElements()
                 $paymentSlipData->getPayerLine3(),
                 $paymentSlipData->getPayerLine4()
             ];
-            $elements['payerLeft'] = [
-                'lines' => $lines,
+            // need to separate, as left payer lines are much shorter
+            $leftLines = $lines;
+            // modify the array containing the left payer lines
+            foreach ($leftLines as $index => $line) {
+                $wrapString = wordwrap($line, 30, "\n", 38);
```

Comment:
Those two literals ``30`` and ``38`` are magic numbers. They should either be saved in self explaining variables or class constants.
May be they could even be configured/settable.